### PR TITLE
doc: contributing: add an 'other guidelines' section for soft guidelines

### PR DESCRIPTION
**Pull request by nodiscc**
- these guidelines are only relevant for a small fraction of additions but would significantly increase the number of requirements in the pull request template, so add them in a separate document
- move the guideline about providing a list of differences for forks to this document
- ref. https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues/33#issuecomment-1886405382

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -84,7 +84,17 @@ To do this from Github's web interface:
...
-### Other
+### Other guidelines
+
+In addition to guidelines listed in the [Pull Request template](.github/PULL_REQUEST_TEMPLATE.md), these general rules help keep the list consistent:
+- Please avoid redundant terms in project descriptions, such as _open-source_, _free_, _self-hosted_... as their presence on awesome-selfhosted already implies this.
+- Prefer shorter forms for descriptions - for example, `Minimalist text adventure game` would be preferred to `A minimalist text adventure game` or `$PROJECT is a minimalist text adventure game`).
+- If the the project has no documentation in English, please add `(documentation in $LANGUAGE)` at the end of the description.
+- If the project is presented as an alternative to another service or application, please mention it as `(alternative to $PRODUCT1, $PRODUCT2)` at the end of the description.
+- If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
+
```
Comment (Rabenherz112) [inline CONTRIBUTING.md:95]:
```suggestion
- Projects which are forked from other projects must include `(fork of $PROJECT)` in the end of their descriptions.
- Software that distributes a single static binary should indicate the programming language it is written in

```

**Review by Rabenherz112 — COMMENTED** [review]
I would have another two additions. One would for forked projects to actually include that 'forked of' part in their description and that if a project offers binary that the orginal language can be used in platforms. (This might need some rephrasing for clarity)

File: CONTRIBUTING.md (hunk 1 above)
Comment (kokomo123) [reply CONTRIBUTING.md:95]: I agree with this as it does make it more clearer to the user on how to handle contributing.

**Review by kokomo123 — APPROVED** [review]
LGTM as all the necessary changes have been made. This does make sense for further explaination of the guidelines, and clarifying it a bit further for newcomers.

**Review by Rabenherz112 — APPROVED** [review]

**Pull request merged**